app/resources/sparql/basic.sparql

```diff
@@ -216,16 +216,20 @@ WHERE
         BIND(iri(REPLACE(?dcterm_xsi_type, "edtf:", "http://id.loc.gov/datatypes/edtf/")) AS ?edtf_dt)
 
         BIND(
-            IF(
-                bound(?dcterm_xsi_type), 
-                STRDT(?dcterm_value, ?edtf_dt), 
-                IF(
-                    bound(?dcterm_xml_lang), 
-                    STRLANG(?dcterm_value, ?dcterm_xml_lang), 
+            IF(?dcterm_p = dct:available, xsd:dateTime(?dcterm_value),
+                IF(?dcterm_p = dct:extent, xsd:time(?dcterm_value),
```

Comment:
Dit voegt het type bij default toe. Willen we liever dat het type gedefinieerd is in de dctrms XML?

Reply:
nee, default is goed want het profiel laat niets anders toe. Maar: als er een verkeerde `xsi:type` wordt meegegeven, dan moet je wel een validatiefout krijgen en dat doet je query momenteel niet

Reply:
maw, die IF zit te hoog; moet na 
```
                        bound(?dcterm_xsi_type),
                        STRDT(?dcterm_value, ?edtf_dt),
```